lib/po_libc_wrappers.c

```diff
@@ -199,6 +200,26 @@ open(const char *path, int flags, ...)
     return _open(path, flags, mode);
 }
 
+/**
+ * Capability-safe wrapper around the `rename(2)` system call.
+ *
+ * `rename(2)` accepts a path argument that can reference the global filesystem
+ * namespace. This is not a capability-safe operation, so this wrapper function
+ * attempts to look up the path (or a prefix of it) within the current global
+ * po_map and converts the call into the capability-safe `renameat(2)` if
+ * possible. If the current po_map does not contain the sought-after path,
+ * this wrapper will call `renameat(AT_FDCWD, original_path, ...)`, which is
```

Comment:
i.e., we'll return `ECAPMODE` _Not permitted in capability mode_ if we do not have a suitable path in `po_map`. Sounds good.